lib/passport/callbacks.coffee

```diff
@@ -132,7 +132,7 @@ onAccessToken = (req, done, params) -> (err, res) ->
   else if msg.match('no account linked')?
     if (req?.session? && params?)
       { sign_up_intent, receive_emails, accepted_terms_of_service } = req.session
-      params = { ...params, ...{ sign_up_intent, receive_emails, accepted_terms_of_service } }
+      _.extend(params, { sign_up_intent, receive_emails, accepted_terms_of_service })
```

Comment:
@mzikherman @erikdstock I believe this achieves the same thing we were trying to do with the spread, but wanted to double check with you guys.

Reply:
Yup looks good! `Object.assign` would prob work as well in Force